modules/installation-launching-installer.adoc

```diff
@@ -212,7 +222,24 @@ and compute nodes.
 .. Select the base domain to deploy the cluster to. All DNS records will be
 sub-domains of this base and will also include the cluster name.
 endif::osp[]
+ifdef::vsphere[]
+.. Select *vsphere* as the platform to target.
+.. Specify the name of your vCenter instance.
+.. Specify the user name and password for the vCenter account that has the required permissions to create the cluster.
++
+The installation program connects to your vCenter instance.
+.. Select the datacenter in your vCenter instance to connect to.
+.. Select the default vCenter datastore to use.
+.. Select the vCenter cluster to install the {product-title} cluster in.
+.. Select the network in the vCenter instance that contains the virtual IP addresses and DNS records that you configured.
+.. Enter the virtual IP address for that you configured for control plane API access.
```

Comment:
Looks like an extra word "for" in here.
"Enter the virtual IP address **for** that you configured for control plane API access."

Same with the line below.

Reply:
Looks like 7 times if you do ctrl-f on the page. I'll call them out.